graph/converter.py

```diff
@@ -0,0 +1,350 @@
+import graph.oval_graph
+import collections
+import re
+import uuid
+
+
+class converter():
+    def __init__(self, tree):
+        self.VALUE_TO_BOOTSTRAP_COLOR = {
+            "true": "text-success",
+            "false": "text-danger",
+            "error": "text-dark",
+            "unknown": "text-dark",
+            "noteval": "text-dark",
+            "notappl": "text-dark"
+        }
+
+        self.VALUE_TO_ICON = {
+            "true": "glyphicon glyphicon-ok",
+            "false": "glyphicon glyphicon-remove",
+            "error": "glyphicon glyphicon-question-sign",
+            "unknown": "glyphicon glyphicon-question-sign",
+            "noteval": "glyphicon glyphicon-question-sign",
+            "notappl": "glyphicon glyphicon-question-sign"
+        }
+
+        self.VALUE_TO_HEX_COLOR = {
+            "true": "#00ff00",
+            "false": "#ff0000",
+            "error": "#000000",
+            "unknown": "#000000",
+            "noteval": "#000000",
+            "notappl": "#000000"
+        }
+
+        if isinstance(tree, graph.oval_graph.OvalNode):
+            self.tree = tree
+        else:
+            raise ValueError('err - this is not tree created from OvalNodes')
+
+    def _get_node_icon(self):
+        values = self._get_node_style()
+        return dict(
+            color=self.VALUE_TO_BOOTSTRAP_COLOR[values['negation_color']],
+            icon=self.VALUE_TO_ICON[values['out_color']],
+        )
+
+    def get_comment(self):
+        if self.tree.comment is not None:
+            return str(self.tree.comment)
+        return ""
+
+    def to_JsTree_dict(self):
```

Comment:
Make a JSTree converter class that would contain a `to_dict` method, and a Sigma converter class with another `to_dict` method. It would be best to have a base `Converter` class, and those `JSTree` and `Sigma` would derive from it.

Reply:
I plan to remove `SigmaJs` from the project. It is therefore not necessary to create inheritance classes.